src/contracts/pods/EigenPod.sol

```diff
@@ -605,8 +614,8 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         address recipient,
         uint64 withdrawalAmountGwei,
         ValidatorInfo memory validatorInfo
-    ) internal {
-        uint256 amountToSend;
+    ) internal returns(WithdrawalInfo memory) {
```

Comment:
imo the struct is bulky and not needed.

returning uint256,int256 is better

Reply:
stack too deep ma boi